### DISCUSSION TITLE
Instructions to Install Marsupial EMR

### DISCUSSION THREAD

**Discussion by seddinger** [marsupialproject/marsupial-emr#3] [General]
Is there a link to step by step instructions on how to install Marsupial EMR?

**Comment by marsupialproject** [comment]
There's no installer, just download the release, unzip it to a folder, and run the program. The example databases are attached to the release page. You'll want to download those too.

https://github.com/marsupialproject/marsupial-emr/releases

If this is not clear enough let me know.

**Reply by seddinger** [reply]
So, Marsupial EMR is not a web based program, it can only run from the "Marsupial.exe" file correct?

Thanks, for the reply!!

**Reply by marsupialproject** [reply]
Yep that's right. It's designed for one user to access it at a time on the local computer.

**Comment by seddinger** [comment]
Hi, how do fix this error?  We have Marsupial EMR setup on the shared sever with our DB files.  Now we're getting this error when trying to start the server from the local machine.  Not for sure how to configure Marsupial.exe.config, Marsupial Editor.exe.config, and Marsupial Instructor.exe.config files.  Any help would be nice.

Is there any examples that you can show us for the three config files?

![Capture](https://github.com/marsupialproject/marsupial-emr/assets/82035934/cb7e89c5-d1a2-41ea-9af5-cfd9a4f7a1cf)

**Reply by marsupialproject** [reply]
It's a permissions issue with Windows not letting Marsupial have network access for the current user. Enter this in a command prompt (you may need to run the command prompt as administrator):

netsh http add urlacl url=http://+:52101/ user=Everyone listen=yes

This will give permission to programs to access port 52101, the port Marsupial uses for network communication.
This will get rid of the error, but you may also have to manually open port 52101 UDP/TCP in Windows Firewall for communication over the network to work.

The config files are not editable.

**Reply by marsupialproject** [reply]
And be careful running Marsupial from a shared server. The program does not support multiple instances or multiple users accessing the same database.

**Reply by seddinger** [reply]
Thank you!!

**Comment by seddinger** [comment]
Not for sure why this is happening, when trying to connect the marsupial editor to the our current DB that we are running, the DB crashes.  There's data in the DB but not for sure why it crashes when we go to connect it to edit the patient files.  How do you fix this issue?

**Reply by marsupialproject** [reply]
I haven't had that happen. If you send the DB to me I can take a look at it.

marsupialemr at gmail.com

**Reply by marsupialproject** [reply]
followup: fixed by redownloading Editor. Possible configuration error

**Comment by seddinger** [comment]
Sorry, my fault I fixed it!!

**Reply by marsupialproject** [reply]
Good to hear. I'm closing this thread since the install problems were solved. If you have anymore unrelated issues please make a new thread.

**Comment by seddinger** [comment]


**Reply by marsupialproject** [reply]
Please make a new thread for issues unrelated to installation

**Comment by eddingers** [comment]
Hi,
Could you be more specific on what you mean by "Make a new thread if you're having a new issue"?
Thanks,
Samuel Eddinger

**Comment by eddingers** [comment]
Hi, Sorry I know what you mean now.

How do you fix this error, I get when trying to connect Instructor with the EMR?  See the attached error.

![Marsupial Instructor Error](https://github.com/user-attachments/assets/5ba3b662-0829-42d7-b8a9-fe8e27f7c1dc)

Thanks,
Samuel Eddinger
